alembic/env.py

```diff
@@ -16,6 +16,8 @@
 # add your model's MetaData object here
 # for 'autogenerate' support
 sys.path.insert(0, path.realpath(path.join(path.dirname(__file__), '..')))
+# This path is purely for alembic to work on the packaged application
+sys.path.insert(1, "/opt/venvs/securedrop-client/lib/python3.5/site-packages")
```

Comment:
Why is this necessary? Shouldn't we keep the alembic directory in a fix position relative to the app? The first path insertion should catch every case, no?

Reply:
Because this is how it can help in packaging. We need static config files to able to package them easily and without any dynamic magic at runtime.

Reply:
I still don't understand. What is the exact case that you're describing?

Reply:
The first path is for the development environment, and the second path is for the case where we are running using Debian package. The `alembic` directory is installed in the standard `/usr/share/securedrop-workstation/` location.

Reply:
Why isn't the alembic directory placed somewhere relative to the source code? Wouldn't that be simpler?